examples/custom-renderer-codelab/src/renderers/javascript.js

```diff
@@ -1,16 +1,6 @@
-/**
- * @license
- * Copyright 2020 Google LLC
- * SPDX-License-Identifier: Apache-2.0
- */
-
-/**
- * @fileoverview The full custom renderer built during the custom renderer
- * codelab, in ES6 syntax.
- * @author fenichel@google.com (Rachel Fenichel)
- */
-
-class CustomConstantsProvider extends Blockly.blockRendering.ConstantProvider {
+import * as Blockly from 'blockly/core';
```

Comment:
Add license and fileoverview back. Author tag is optional now.

Reply:
should https://github.com/google/blockly-samples/blob/master/examples/custom-generator-codelab/src/generators/json.js and https://github.com/google/blockly-samples/blob/master/examples/custom-generator-codelab/src/blocks/json.js have fileoverview's too?

Reply:
i've updated this one. leaving unresolved to find out about my question (i will put a fileoverview on those, if so)

Reply:
(in a separate PR, to be clear!)

Reply:
Yes, please add fileoverviews there too. At this point I consider them not required, but very nice to have for anyone who is exploring sample code.